mne/io/brainvision/brainvision.py

```diff
@@ -849,3 +844,10 @@ def __call__(self, description):
             code = (super(_BVEventParser, self)
                     .__call__(description, offset=_OTHER_OFFSET))
         return code
+
+
+def _check_bv_annot(descriptions):
+    markers_basename = set([dd.rstrip('0123456789 ') for dd in descriptions])
+    bv_markers = (set(_BV_EVENT_IO_OFFSETS.keys())
+                  .union(set(_OTHER_ACCEPTED_MARKERS.keys())))
+    return len(markers_basename - bv_markers) == 0
```

Comment:
We could be even stricter here if I understand correctly that the numeric code ending a BV code is always 3 characters (whitespace left padded plus an int 0-255), i.e., 

`set([dd[:-3] for dd in descriptions])`

Reply:
@cbrnr this is the case right?

Reply:
I think so, at least I've only ever seen markers like that (`S  2`, `R  1`, ...).

Reply:
I started by `set([dd[:-3] for dd in descriptions])` but I went for `rstrip` because I found it more readable. But either one is fine.